src/SFA.DAS.EmployerAccounts.Web/Controllers/HomeController.cs

```diff
@@ -140,9 +172,21 @@ public ActionResult WhatYoullNeed()
 
         [HttpPost]
         [Route("whatYoullNeed")]
-        public ActionResult WhatYoullNeedPost()
+        public ActionResult WhatYoullNeed(int? choice)
         {
-            return RedirectToAction(ControllerConstants.RegisterUserActionName);
+            switch (choice ?? 0)
+            {
+                case 2: return RedirectToAction(ControllerConstants.RegisterUserActionName);
```

Comment:
Does the case where the user has answered 'No' not also need to return as error?

Reply:
It will as anything other than yes uses the default route.

Reply:
Not sure about the redirect as it’s not yet exposed in the current journey so I assume this can be looked into once we use the page.

Reply:
> It will as anything other than yes uses the default route.

Ahh, sorry I assumed '2' was 'No' thinking '1', '2' going down in list, but 'obviously' that list goes in reverse order.

Reply:
> Not sure about the redirect as it’s not yet exposed in the current journey so I assume this can be looked into once we use the page.

Yes, I did wonder about that, but can it be tested (via direct URL) without the re-direct?

Reply:
Thinking about it, we should not have to put new page re-directs in; iirc.